README.md

```diff
@@ -22,12 +33,134 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
 - Distributed training support
 - TensorBoard support
 
+See [How it works](#how-it-works) for more details.
+
+## Support
+
+### [Latest release v0.7.2](https://github.com/awslabs/sagemaker-debugger/releases)
+
+   - Introducing experimental support for TF 2.x training scripts using GradientTape -
+   With this update, weights, bias, loss, metrics, and gradients are captured by SageMaker Debugger.
+   GradientTape in TF 2.x captures these tensors from custom training jobs. An example of GradientTape implementation to a custom ResNet training script using TensorFlow's Keras interface is provided at [tf_keras_gradienttape.py](https://github.com/awslabs/sagemaker-debugger/blob/master/examples/tensorflow2/scripts/tf_keras_gradienttape.py).
+   GradientTape does not work with Zero Script Change experience at this time.
+
+        *Note*: Training scripts using GradientTape for higher-order gradients or multiple tapes are not supported. Distributed training scripts that use GradientTape are not supported at this time.
+
+   - Support `SyncOnReadVariable` in mirrored strategy - Fixes a bug that occurred because the `SyncOnRead` distributed variable was not supported with `smdebug`. Also enables the use of `smdebug` with training scripts using TF 2.x MirroredStrategy with the `fit()` API.
+
+   - Turn off hook and write only from one worker for unsupported distributed training techniques – Fixes a crash when distributed training in PyTorch framework is implemented using generic multiprocessing library, which is not a method supported by `smdebug`. This fix handles this case and ensures that tensors are saved.
+
+   - Bug fix: Pytorch: Register only if tensors require gradients – Users were observing a crash when training with pre-trained embeddings which does not need gradient updates. This fix checks if a gradient update is required and registers a backward hook only in those cases.
+
+### Setting up SageMaker Debugger
+
+`smdebug` library runs on Python 3.x. Install `smdebug` through:
+
+```
+pip install smdebug
+```
+
+### Supported Frameworks
+There are two ways in which you can enable SageMaker Debugger while training on SageMaker&mdash;Zero Script Change and Bring Your Own Training Container.
```

Comment:
```suggestion
There are two ways in which you can enable SageMaker Debugger while training on SageMaker&mdash;Zero Script Change and Bring Your Own Training Container (BYOC).
```
Does training need to be in there? We use BYOC in a lot of places... and it might be easier to reference this way.